modules/nw-ingress-gateway-api-manage-succession.adoc

```diff
@@ -0,0 +1,59 @@
+// Modules included in the following assemblies:
+//
+// * networking/configuring_ingress_cluster_traffic/ingress-gateway-api.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="nw-ingress-gateway-api-manage-succession_{context}"]
+= Preparing for Gateway API management succession by the Ingress Operator
+
+Starting In {product-title} 4.19, the Ingress Operator manages the lifecycle of any Gateway API custom resource definitions (CRDs). Upgrading from a version before 4.19 of {product-title} where this management was not present requires you to replace or remove any Gateway API CRDs that already exist in the cluster so that they conform to the specific {product-title} specification required by the Ingress Operator. {product-title} version 4.19 requires Gateway API Standard version 1.2.0 CRDs.
+
+[WARNING]
+====
+Updating or deleting Gateway API resources can result in downtime and loss of service or data. Be sure you understand how this will affect your cluster before performing the steps in this procedure. If necessary, back up any Gateway API objects in YAML format in order to restore it later.
+====
+
+.Prerequisites
+. Install the OpenShift CLI (`oc`).
```

Comment:
Just a suggestion to reword this prereq to make it more parallel with the others:
```suggestion
. You have installed the OpenShift CLI (`oc`).
```